APIScrape/scrape.php

```diff
@@ -81,8 +81,7 @@ function fillCatBreedToDB(PDO $db, array $catBreeds)
 {
     foreach($catBreeds as $id => $breed) {
         $sql = $db->prepare('INSERT INTO `breed` (breed) VALUES (:breedToString);');
-        $breedToString = "$breed";
-        $sql->bindParam('breedToString', $breedToString, PDO::PARAM_STR);
+        $sql->bindParam('breedToString', $breed, PDO::PARAM_STR);
```

Comment:
not blocking merge, but this should just be `'breed'`, not `'breedToString`

Reply:
Ah, reverted, nvm

Reply:
Seems to have been implemented the same way again. `: breedToString` -> `breed` pls

Reply:
on it again